src/Immunisatie-0-1/Cert/PHR-Client/imm-retrieve-1-1.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://hl7.org/fhir/STU3/testscript.sch" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TestScript xmlns="http://hl7.org/fhir" xmlns:nts="http://nictiz.nl/xsl/testscript"
+    nts:scenario="client">
+    <id value="imm-retrieve-1-1"/>
+    <version value="r4-1.0.0"/>
+    <name value="Immunization - Client - Scenario 1.1 - Retrieve Immunization resources of van Putten"/>
+    <description value="Scenario 1.1 - Retrieve Immunization resources of van Putten"/>
+    <nts:patientTokenFixture href="imm-Patient-01-token.xml"/>
+
+    <test>
+        <name value="Scenario 1.1 - Retrieve Immunization resources"/>
+        <description
+            value="Query Immunization resource including the patient"/>
+        <nts:include value="medmij/test.phr.search" scope="common"
+            resource="Immunization"/>
```

Comment:
Dit test niet 'including the patient'. Volgens mij mist hier in dat geval de 'params' parameter in de include.

Reply:
Of je moet de description aanpassen natuurlijk

Reply:
"Dat ligt aan de IG" is het goede antwoord denk ik. Als het daar niet staat dan kunnen we het inderdaad niet verwachten
Maar je doet verderop nog wel includes van zorgverlener/zorgaanbieder dacht ik? Staan die wel in de IG dan?

Reply:
De includes staan in de IG.

Reply:
Ik heb het uit de descriptions gehaald